core/tests/test_handlers.py

```diff
@@ -637,6 +637,57 @@ async def handler(*args, **kwargs):
     }
 
 
+async def test_weakrefcallable_bound_method():
```

Comment:
These tests duplicate the ones that were added when the GTK4 backend migrated the weakrefhandler.